src/js/actions/layereffects.js

```diff
@@ -408,6 +410,8 @@ define(function (require, exports) {
             coalesce: false
         };
 
+        this.dispatch(events.style.HIDE_HUD);
```

Comment:
I think the rule now is that dispatches must use `dispatchAsync` *unless* it is already inside a promise chain.  @iwehrman correct me if I'm misremembering our recent conversation.